typescript/lib/parsers/openai.ts

```diff
@@ -851,6 +906,28 @@ function multiChoiceMessageReducer(
   return messages;
 }
 
+function buildOutputData(
+  message: Chat.ChatCompletionMessageParam | null
+): OutputDataWithValue | string | undefined {
+  let outputData: OutputDataWithValue | string | undefined = undefined;
+  if (message !== null) {
+    if (message.content !== null) {
```

Comment:
`message.content != null`

Reply:
message.content can only be null in all implementations of `ChatCompletionMessageParam`, but yea sure I can future proof